README.md

```diff
@@ -1,199 +1,263 @@
 Python for OpenShift - Docker images
-========================================
+====================================
 
 This repository contains sources of the images for building various versions
 of Python applications as reproducible Docker images using
-[source-to-image](https://github.com/openshift/source-to-image).
-User can choose between RHEL and CentOS based builder images.
+[Source-to-Image](https://github.com/openshift/source-to-image).
+User can choose between CentOS and RHEL based builder images.
 The resulting image can be run using [Docker](http://docker.io).
 
 
 Versions
----------------
-Python versions currently provided are:
-* python-3.3
+--------
 
-RHEL versions currently supported are:
-* RHEL7
+Python versions currently provided:
+* Python 3.3
 
-CentOS versions currently supported are:
-* CentOS7
+CentOS versions currently supported:
+* CentOS 7
 
+RHEL versions currently supported:
+* RHEL 7
 
-Installation
----------------
-To build Python image, choose between CentOS or RHEL based image:
-*  **RHEL based image**
 
-    To build a rhel-based python-3.3 image, you need to run the build on a properly
-    subscribed RHEL machine.
+Image name structure
```

Comment:
Please don't change the organization of this file, unless you plan to do it for all of the images. We have them quite identical for ease of maintenance and readability. In other images Image name structure is just before the envs description.